docs/development/setup_development.rst

```diff
@@ -39,6 +39,11 @@ be installed via the official documentation links:
 .. _`Docker CE for Ubuntu`: https://docs.docker.com/install/linux/docker-ce/ubuntu/
 .. _`Docker CE for Debian`: https://docs.docker.com/install/linux/docker-ce/debian/
 
+.. warning::
+
+    The official documentation currently misses the detail that the user needs
```

Comment:
By "official documentation" do you mean SecureDrop's or Docker's? Additionally, all commands can be `sudo`'d which is what Docker recommends since having a user in the `docker` group is actually a security hole as it effectively means commands can be run as root.

Reply:
Aha... I meant dockers docs. 

I didn't see the instructions for using `sudo` in their docs and the example commands I read in both docker and SecureDrop suggest (to me) that `sudo` wasn't expected, hence this change. However, please disregard this change given the security concerns of being in the `docker` group. In any case, would be good to know what advice to give (I was just logging this as I worked my way through the instructions). Thanks!